packages/frontend/src/pages/Profile/Notifications/Notifications.tsx

```diff
@@ -21,7 +126,174 @@ export const Notifications = () => {
+                  linkIcon
+                />
+              </>
+            )}
+          </List>
+        </Settings>
+        {settings && <Heading size="lg">{t('profile.notifications.heading_per_actor')}</Heading>}
+        <AccountSettings actors={dummy as Array<SettingsItemProps>} />
+      </Section>
     </PageBase>
   );
 };
+
+type AlertSettings = {
+  [id: string]: {
+    smsAlerts: boolean;
+    emailAlerts: boolean;
+  };
+};
+
+export const AccountSettings = ({ actors }: { actors: Array<SettingsItemProps> }) => {
+  const { t } = useTranslation();
+  const [expanded, setExpanded] = useState<string | undefined>(undefined);
+  const [alertSettings, setAlertSettings] = useState<AlertSettings>({});
+
+  const handleAlertChange = (id: string, alerts: { smsAlerts: boolean; emailAlerts: boolean }) => {
+    setAlertSettings((prev) => ({
+      ...prev,
+      [id]: alerts,
+    }));
+  };
+
+  return (
+    <Settings>
+      <List>
+        {actors.map((item, index) => {
+          const alerts = alertSettings[item.id!] || { smsAlerts: false, emailAlerts: false };
+          let badgeLabel: string | undefined;
+          if (alerts.smsAlerts && alerts.emailAlerts) {
+            badgeLabel = `SMS ${t('word.and')} ${t('profile.landing.email')}`;
+          } else if (alerts.smsAlerts) {
+            badgeLabel = 'SMS';
+          } else if (alerts.emailAlerts) {
+            badgeLabel = t('profile.landing.email');
+          } else {
+            badgeLabel = undefined;
+          }
+
+          return (
+            <Fragment key={item.id}>
+              {index > 0 && <Divider />}
+              <SettingsItem
+                icon={item.icon}
+                as="button"
+                title={item.title}
+                description={item.description}
+                badge={badgeLabel && { label: badgeLabel }}
+                collapsible
+                onClick={() => setExpanded(expanded === item.id ? undefined : item.id)}
+                expanded={expanded === item.id}
+              >
+                <AccountDetails id={item.id!} onAlertChange={handleAlertChange} initial={alerts} />
+              </SettingsItem>
+            </Fragment>
+          );
+        })}
+      </List>
+    </Settings>
+  );
+};
+
+type AccountDetailsProps = {
+  id: string;
+  onAlertChange: (id: string, alerts: { smsAlerts: boolean; emailAlerts: boolean }) => void;
+  initial: {
+    smsAlerts: boolean;
+    emailAlerts: boolean;
+  };
+};
+
+const AccountDetails = ({ id, onAlertChange, initial }: AccountDetailsProps) => {
+  const { t } = useTranslation();
+  const [smsAlerts, setSmsAlerts] = useState(initial.smsAlerts);
+  const [emailAlerts, setEmailAlerts] = useState(initial.emailAlerts);
+  const [phone, setPhone] = useState('');
+  const [email, setEmail] = useState('');
+
+  // biome-ignore lint/correctness/useExhaustiveDependencies: Full control
+  useEffect(() => {
```

Comment:
I think we should avoid using `useEffect` here, instead call onAlerthange on `onChange` for onChange on the switches